CHANGELOG/CHANGELOG-1.5.1.md

```diff
@@ -0,0 +1,16 @@
+# v1.5.1 
+
+## Changelog since v1.5.0 
+
+## Changes by Kind 
+
+### Features 
+
+- CSM 1.5.1 release specific changes. ([#582](https://github.com/dell/csm/issues/582))
+- Show volumes associated with the tenant from the k8s server. ([#408](https://github.com/dell/csm/issues/408))
+- CSI PowerScale documentation misses the privileges to user to use CSM replication. ([#516](https://github.com/dell/csm/issues/516))
+
+### Bugs 
+
+- PVC fails to resize with the message spec.capacity[storage]: Invalid value: "0": must be greater than zero. ([#507](https://github.com/dell/csm/issues/507))
```

Comment:
this bug also seems to have an incorrect milestone, this is not planned for 1.5.1.

Reply:
I hope this was not because 1.6.0 getting renamed to 1.5.1 at that time.

Reply:
I've updated the milestone to 1.6.0 for #507.

Reply:
> this bug also seems to have an incorrect milestone, this is not planned for 1.5.1.

Reply:
> I hope this was not because 1.6.0 getting renamed to 1.5.1 at that time.

Reply:
> I've updated the milestone to 1.6.0 for #507.

Reply:
> > this bug also seems to have an incorrect milestone, this is not planned for 1.5.1.
> 
>

Reply:
> > > this bug also seems to have an incorrect milestone, this is not planned for 1.5.1.
> > 
> > 
> 
>

Reply:
> > > > this bug also seems to have an incorrect milestone, this is not planned for 1.5.1.
> > > 
> > > 
> > 
> > 
> 
>